app/lib/azul_search_service.rb

```diff
@@ -73,8 +76,12 @@ def self.get_results(selected_facets:, terms:)
       result[:facet_matches] = get_facet_matches(entry_hash, merged_facets)
       if terms
         # only store result if we get a text match on project name/description
-        result[:term_matches] = get_search_term_weights(result, terms)
-        results[short_name] = result if result.dig(:term_matches, :total) > 0
+        match_info = get_search_term_weights(result, terms)
+        if match_info[:total] > 0
+          result[:term_matches] = match_info[:terms].keys
+          result[:term_search_weight] = match_info[:total]
```

Comment:
A name like `term_weights` seems more intuitive to follow, and more congruent with the function name.

```suggestion
        term_weights = get_search_term_weights(result, terms)
        if term_weights[:total] > 0
          result[:term_matches] = term_weights[:terms].keys
          result[:term_search_weight] = term_weights[:total]
```